javascript/game.js

```diff
@@ -1,117 +1,69 @@
+$(document).ready(function () {
+	var orange = $('#orange');
+	var cherry = $('#cherry');
+	var banana = $('#banana');
+	var berry = $('#berry');
 
-// random number to try and add too
+	var gameWin;
+	var total;
 
-var gameWin = Math.floor(Math.random() * 120) + 19;
+	var win = 0;
+	var loss = 0;
 
-// display gameWin in #gameNumber div
-$("#gameNumber").html(gameWin);
+	function resetGameNumbers () {
```

Comment:
This is all logic that you were repeating at the end of any round; And whenever you're repeating code it's a great opportunity to refactor it into one function so as to keep things DRY.